github.md

```diff
@@ -37,3 +46,50 @@ For example, when configuring [auto-publishing for an npm library](./npm-package
 | production | Repositories that are deployed to production                       |
 
 If none of the above topics fit your need, a PR should be opened to add the new topic to this list before use.
+
+## Repository contents
+
+There are three tiers of information, defined below. In short:
+  - Never commit secret information
+  - Avoid private information in public repositories
+
+### Public
+Things we could happily put on the front page of the Guardian.
+
+Examples:
+  - source code
+  - diagrams
+  - architecture decision records
+
+### Private
+Things we do not want to be common knowledge, but knowing them does not directly compromise anything.
+
+Generally this means that knowing that detail would make an exploit (technical or people-wise) easier to achieve, especially when combining a few of them.
+
+Not for public repositories. Fine for private repositories.
+
+Examples:
+  - AWS account IDs
+  - S3 bucket names
+
+### Secret
+Information that directly causes problems! These should be kept out of VCS completely.
+
+Secrets should be rotated regularly.
+
+If leaked, treat it as a security incident.
```

Comment:
Presumably it IS a security incident?

"If leaked, it is a security incident, follow the ..."

Reply:
👍🏽  updated.